GHzDACs/Cleanup/dac.py

```diff
@@ -936,8 +962,252 @@ class DacRunner_Build11(DacRunner_Build7):
 class DAC_Build11(DAC_Build7):
     RUNNER_CLASS = DacRunner_Build11
 
+
 fpga.REGISTRY[('DAC', 11)] = DAC_Build11
 
+
+### Jump table
+
+class DacRunner_Build13(DacRunner_Build7):
+    # noinspection PyMissingConstructor
```

Comment:
pycharm complains when you don't call the superclass' constructor. But we don't want to do that here, because the JT stuff is different enough. Which maybe indicates that this isn't the proper class structure...